aes-kw/src/kek.rs

```diff
@@ -36,6 +39,18 @@ pub const IV_LEN: usize = SEMIBLOCK_SIZE;
 /// ```
 pub const IV: [u8; IV_LEN] = [0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6, 0xA6];
```

Comment:
Should this be renamed to `KW_IV`? I didn't want to break backwards compatibility.